<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>cmr10</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt; font-style:normal; color:#800002;&quot;&gt;PENCIL PRODUCER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="makePencil">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(175, 127, 51);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Make Pencil</string>
    </property>
   </widget>
   <widget class="QLabel" name="inventory">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(233, 185, 110);
background-color: rgb(229, 227, 231);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(193, 125, 17);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Wood:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(186, 78, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Graphite:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="wood">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="graphite">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="balance">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 227, 231);
color: rgb(119, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 227, 231);
color: rgb(101, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#730000;&quot;&gt;Bank Balance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyWood">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 125, 17);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Buy Wood</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyGraphite">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 78, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Buy Graphite</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 227, 231);
color: rgb(0, 72, 57);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Price of Pencil:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="pencilPrice">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 227, 231);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="increasePencil">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(218, 211, 0);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QToolButton" name="decreasePencil">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(218, 211, 0);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 227, 231);
color: rgb(0, 72, 57);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Public demand:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="publicDemand">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 227, 231);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buyApm">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 142, 144);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Buy APM</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberofApm">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(143, 142, 144);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;APM's:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="priceofApm">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 142, 144);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 227, 231);
color: rgb(0, 72, 57);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#0a574a;&quot;&gt;Pencils Produced:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 227, 231);
color: rgb(192, 181, 0);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#cabe19;&quot;&gt;Total pencils produced:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalinventory">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 227, 231);
border-color: rgb(233, 185, 110);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rateofApm">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(143, 142, 144);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Rate of APM:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="priceofWood">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 125, 17);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="priceofGraphite">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 78, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="intelligenceLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(92, 161, 193);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Intelligence:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="intelligence">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upgradeApm">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(218, 211, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>UPGRADE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upgradeMarketing">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 112, 142);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Upgrade Marketing</string>
    </property>
   </widget>
   <widget class="QLabel" name="valueofMarketing">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="debugtool">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug Tool</string>
    </property>
   </widget>
   <widget class="QPushButton" name="debugBalance">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 161, 193);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Bank Balance + 1000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="debugPencils">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 161, 193);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pencils Produced + 1000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="debugTotalPencils">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 161, 193);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Total Pencils Prod.+ 1000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="debugWoodGraphite">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>250</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 161, 193);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Wood+10, Graphite+10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(92, 161, 193);</string>
    </property>
    <property name="text">
     <string>Marketing unlocked?</string>
    </property>
   </widget>
   <widget class="QLabel" name="unlockedMarketing">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="incINT">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>300</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 161, 193);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Increase Intelligence +10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="incMAR">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>350</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 161, 193);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Increase Marketing +3</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>480</y>
      <width>227</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(163, 43, 0);
color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>QUIT</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="loadState">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(218, 211, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>LOAD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="saveState">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(34, 135, 93);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>SAVE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="info">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 227, 231);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>buyApm</zorder>
   <zorder>label</zorder>
   <zorder>makePencil</zorder>
   <zorder>inventory</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>wood</zorder>
   <zorder>graphite</zorder>
   <zorder>balance</zorder>
   <zorder>label_6</zorder>
   <zorder>buyWood</zorder>
   <zorder>buyGraphite</zorder>
   <zorder>label_5</zorder>
   <zorder>pencilPrice</zorder>
   <zorder>increasePencil</zorder>
   <zorder>decreasePencil</zorder>
   <zorder>label_7</zorder>
   <zorder>publicDemand</zorder>
   <zorder>numberofApm</zorder>
   <zorder>label_8</zorder>
   <zorder>priceofApm</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>totalinventory</zorder>
   <zorder>rateofApm</zorder>
   <zorder>label_12</zorder>
   <zorder>priceofWood</zorder>
   <zorder>priceofGraphite</zorder>
   <zorder>intelligenceLabel</zorder>
   <zorder>intelligence</zorder>
   <zorder>upgradeApm</zorder>
   <zorder>upgradeMarketing</zorder>
   <zorder>valueofMarketing</zorder>
   <zorder>debugtool</zorder>
   <zorder>debugBalance</zorder>
   <zorder>debugPencils</zorder>
   <zorder>debugTotalPencils</zorder>
   <zorder>debugWoodGraphite</zorder>
   <zorder>label_13</zorder>
   <zorder>unlockedMarketing</zorder>
   <zorder>incINT</zorder>
   <zorder>incMAR</zorder>
   <zorder>info</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>makePencil</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setAnimated(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
